.github/workflows/main.yml

```diff
@@ -60,3 +60,16 @@ jobs:
     - name: Test
       run: |
         bundle exec fastlane test
+
+    - name: Test reports upload
+      uses: actions/upload-artifact@master
+      with:
+        name: test-reports
+        path: "fastlane/test_output"
+
+    - name: Tests reports annotations
+      uses: ashley-taylor/junit-report-annotations-action@1.3
```

Comment:
FYI this action is currently broken. This [PR](https://github.com/ashley-taylor/junit-report-annotations-action/pull/16) should fix it.